### PR TITLE
Improve config path discovery message and add an option to configure the ssh.config file

### PR DESCRIPTION
**Pull request by matbu**
Improve message for os-diff config files discovery and add a new option to help to configure the ssh.config:

with yaml format (such as tripleo inventory):
os-diff configure --inventory inventory --output ssh_config --yaml
with txt/ini:
os-diff configure --inventory inventory --output ssh_config 
and from /etc/hosts:
os-diff configure --inventory inventory --output ssh_config --etc

### CODE REVIEW

**Comment by matbu** [comment]
@davidjpeacock If you can take a look at this PR, it's just improving a bit the user experience.
/me working on another one for getting an updated documentation (readme.md)

File: pkg/collectcfg/fetch.go [hunk 1]
```diff
@@ -200,6 +200,13 @@ func PullPodFiles(podId string, containerName string, remotePath string, localPa
 }
 
 func SyncConfigDir(localPath string, remotePath string, sshCmd string, undercloud string) error {
+	// make sure localPath exists
+	mkCmd := "mkdir -p " + localPath
```
Comment (davidjpeacock) [inline pkg/collectcfg/fetch.go]: I don't like this, it's not healthy to Exec or shell out when Go has purpose made idioms.  If we can avoid shelling out we get two benefits among others:

1) We are OS agnostic, the lower level functions will work in different environments from Windows to macOS to BSD to any form of Linux
2) We are not at the mercy of potential security vulnerabilities by blindly trusting external tools on our operator's machine

Example 1:

`os.Mkdirall() // https://pkg.go.dev/os#MkdirAll`

Example 2:

We could make a function:
```
func createDirectory(path string) error {
   // Check if the directory exists
   _, err := os.Stat(path)
   if err == nil {
      return nil
   }
   // If the directory does not exist, create its parent
   if os.IsNotExist(err) {
      err = createDirectory(filepath.Dir(path))
      if err != nil {
         return err
      }
      // Create the directory
      err = os.Mkdir(path, 0777)
      if err != nil {
         return err
      }
   }
   return nil
}
```

**Review by davidjpeacock — CHANGES_REQUESTED** [review]
Mostly great, but one (large) nit pick. :-D

File: pkg/collectcfg/fetch.go (hunk 1 above)
Comment (matbu) [reply pkg/collectcfg/fetch.go]: Thank you for the review, and yes os.Mkdirall() is a best approach.
Fixed in the next commit

**Review by davidjpeacock — APPROVED** [review]

**Pull request merged**